### PR TITLE
Added Ecrc Service class and basic interface.

### PR DESCRIPTION
**Pull request by shaunmillargov**
# Description

Added basic classes for service pattern.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Ran test classes. 

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: **n/a**

### CODE REVIEW

File: src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/EcrcServices.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * @author shaunmillargov
+ *
+ */
+public interface EcrcServices {
+	
+	public String doAuthenticateUser(String accessCode);
+	
+	//TODO - fill in other service method signatures here. 
+	
+}
```
Comment (TayGov) [inline src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/EcrcServices.java]: Missing new lines. Otherwise good.

Comment (shaunmillargov) [reply src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/EcrcServices.java]: Thanks. Added in 2 plcs.

Comment (TayGov) [reply src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/EcrcServices.java]: Exceptions needs it as well.

Comment (shaunmillargov) [reply src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/EcrcServices.java]: Was kind of leaving that for the person who completed the operation. I can add it.

Comment (shaunmillargov) [reply src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/EcrcServices.java]: Done.

**Pull request merged**